ledger/src/blockstore.rs

```diff
@@ -2044,6 +2055,9 @@ impl Blockstore {
                 ),
                 ("num_repaired", num_repaired, i64),
                 ("num_recovered", num_recovered, i64),
+                ("min_turbine_fec_set_count", min_fec_set_count, i64),
+                ("last_turbine_fec_set_count", last_fec_set_count, i64),
+                ("last_turbine_fec_set_size", last_fec_set_size, i64),
```

Comment:
As soon as, we get half of a batch (assuming 32/64), we can reconstruct the batch and if that was the last pending batch in a slot, then the slot is full. So these number will hover around batch-size/2 most of the time, which is not what we are trying to estimate.
so, I am not sure emitting these metrics here is useful.